src/styles/themes/cyberpunk-red.scss

```diff
@@ -0,0 +1,244 @@
+@import "../variables";
+$theme: (
+  //Font
+        font-family: Jost,
+        heading-font-family: Jost,
```

Comment:
(CPRC Dev here) This is probably better as. `var(--cpr-font-primary)` for the `font-family` and possibly `var(--cpr-font-hero)` for the `heading-font-family) if you want to match the system fonts as used in the system (which will also follow if a theme changes the fonts).

Reply:
Done. Was an oversight on my part when moving to the variable structure from my original tinkering.